### ISSUE TITLE
Instead of making a new client from stratch just use Discord's official one

### ISSUE THREAD

**Issue by enescglyn** [SebOuellette/LiveBot#81]
#### Suggestion Type

- Type: Change

- Description: I mean literally you guys treid to make a discord client from stratch just to flex your "BOT" tag on your username, Just build on top of the official discord client, It is easier and better.

#### How would it help?

No need to get used to it, we get all features of discord(not all but i mean features available to bots)

#### Preview of the change

- Screenshots:

- Script:

**Comment by SebOuellette** [comment]
Yeah see, the reason we do it how we do is not to "flex our bot tag". Building on top of discord's client is why livebot alternatives have been shut down, and why we're still up. It's in their TOS and copyright that you cannot modify Discord, or use any part of it as your own. 

This project was also started as a fun way to try and figure out how to build everything discord did, on our own, without the need to reverse engineer Discord (since that's also against TOS).

In short... yeah no this won't change. 

Btw, you added the 'Please use Discord' tag, this means you know that these suggestions should be put in our [discord server](TMUc8Bs), not github.

**Comment by enescglyn** [comment]
I mean you are not forcing me to do so.

**Comment by SebOuellette** [comment]
Uh... We would just prefer that suggestions are put into discord. That way we can keep everything sorted nicely. I'd rather this Github *issues* page be used for just *issues* :)

**Comment by enescglyn** [comment]
By the way there is something called better discord and millions use it, it literally breaks the discord ToS and no one gets banned for it, Their servers have a lot of members i forgot how much but probably more than 20k (more than discord can handle, they made a 2nd server to people join) and i saw someone talking about how discord increased the member limit for their servers.

**Comment by dcrime** [comment]
Yes, correct, Better Discord does break the TOS but there's a catch, they ARE modifying their code BUT did you even see that the plugins and themes get checked? Yeah, they actually get checked and if they're breaking the TOS they don't get added, if they start breaking it then they get removed, if they can't be used anymore then they get removed, easy

We're still sticking to not breaking the TOS and we were contacted by Discord already because they weren't so happy about this project so we made a few changes to satisfy what they didn't like (There was actually nothing breaking the TOS but they were worried about people seeing messages they shouldn't even tho when you add a bot you should be aware already that anything could happen)

**Comment by enescglyn** [comment]
You are talking about message logger, a lot of people use it already and they dont get removed anyway someone maintains a fork of it. Also yes you wont get "support" in betterdiscord server, but in the maintainer's discord, you do.

**Comment by SebOuellette** [comment]
Ok listen. I don't know what your goal is here but we're not going to intentionally break TOS. We have respect for other developers, and understand why TOS exists. I'm locking this discussion cause it's completely pointless.

**Comment by dcrime** [comment]
Ps, be sure to drop by in the [discord server](https://www.discord.gg/TMUc8Bs ) ❤
